docs/docs/Support/supported-software.md

```diff
@@ -0,0 +1,603 @@
+---
+title: Supported software versions
+slug: /support-supported-software
+---
+
+Support covers only the following software versions for Langflow.
```

Comment:
"Support covers" is vague. I think we need to specify that the Luna for Langflow support service provides support for the listed versions. It would also be helpful to link to the page that you will add on docs.datastax.com or to the Luna for Langflow page on datastax.com so that people who don't know about the Luna service and get details.